### PR TITLE
This Repository has been moved to https://github.com/openstack/craton

### PR DESCRIPTION
**Pull request by ahsan518**
This repository has been moved to https://github.com/openstack/craton, which mirrors https://git.openstack.org/cgit/openstack/craton/.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,29 +1,5 @@
...
-
-TODO(all) We plan to move these links to OpenStack infrastructure once
-Craton becomes an OpenStack project.
-
-For information on how to contribute to Craton, please see the
-contents of the `CONTRIBUTING.rst file <CONTRIBUTING.rst>`_.
-
-For more information on Craton distribution license, please see
-the contents of the `LICENSE file <LICENSE>`_.
+## This repository has been moved to https://github.com/openstack/craton, which mirrors https://git.openstack.org/cgit/openstack/craton/. Everything else is nuked
```
Comment (ChristopherMSpencer) [inline README.rst]: I only see one file change, are you nuking the other files?

**Comment by ChristopherMSpencer** [comment]
+1 lgtm

**Comment by sigmavirus24** [comment]
Most projects don't do this. Is there a reason why we're choosing to do it?

**Comment by ChristopherMSpencer** [comment]
Too many people are getting confused about this location and Jim asked Syed to do this yesterday in IRC

**Comment by sigmavirus24** [comment]
I've turned off Travis CI.

**Pull request merged**